modules/CuckooFilter/src/main.rs

```diff
@@ -1,3 +1,57 @@
-fn main() {
-    println!("Hello, world!");
-}
+extern crate redis;
+use redis::Commands;
+use probabilistic_collections::cuckoo::CuckooFilter;
+
+fn main() {
+    let size = std::env::args().nth(1).expect("Please specify the size of the filter");
+    let action = std::env::args().nth(2).expect("Please specify an action");
+    let item = std::env::args().nth(3).expect("Please specify an item");
+    
+    println!("size of the filter {:?}", size);
+
+    let mut cuckoo_filter = CuckooFilter::<String>::new(size.parse::<usize>().unwrap());
+
+    if action == "add"{
+        cuckoo_filter.insert(&item);
+        match add(item) {
+            Ok(_) => println!("Item added"),

```

Comment:
**[Question]** Why are we logging with `println!(...)`? That doesn't seem like its going to end up in the redis logs (correct me if im wrong). Feels like there should a context object that we should be logging to instead.